presto-yarn-package/src/main/slider/package/scripts/configure.py

```diff
@@ -56,6 +61,10 @@ def set_configuration(component=None):
                 shutil.copy2(os.path.join(params.source_plugin_dir, jar), plugin_dir)
 
 
+def _parse_array_and_write(params_array, path):
```

Comment:
I would rename params_array to parameters, as it is called in _store_configuration